src/Component.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class Component extends React.Component {
+    constructor() {
+        super();
```

Comment:
React API expects `props` in super:
```
 constructor(props) {
        super(props);
```
[Docs](https://reactjs.org/docs/react-component.html#constructor)